<!--(C) Copyright Hung-Ching Lin-->

<!--This file is part of jaxa_resources.-->

<!--    qlin_jaxa_resources is free software: you can redistribute it and/or modify-->
<!--    it under the terms of the GNU Lesser General Public License as published by-->
<!--    the Free Software Foundation, either version 3 of the License, or-->
<!--    (at your option) any later version.-->

<!--    qlin_jaxa_resources is distributed in the hope that it will be useful,-->
<!--    but WITHOUT ANY WARRANTY; without even the implied warranty of-->
<!--    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the-->
<!--    GNU Lesser General Public License for more details.-->

<!--    You should have received a copy of the GNU Lesser General Public License-->
<!--    along with qlin_jaxa_resources.  If not, see <http://www.gnu.org/licenses/>.-->

<!--Author:-->
<!--    Hung-Ching Lin (qlin1806@g.ecc.u-tokyo.ac.jp)-->

<!--Contributors (aside from author):-->
<!--    None-->
<launch>
    <node pkg="jaxa_communication" type="delay_teleop_kinematics_transmitter_node.py" name="delay_teleop_transmitter_node" clear_params="true">
        <rosparam subst_value="true">
            robot_kinematic_interfaces_ns: ['/arm1_kinematics', '/arm2_kinematics']
            manipulator_interfaces_ns: ["/escon_4","/escon_2"]
            flip_gripper: [false, false]
            remote_side_open_ports: 21391
        </rosparam>
    </node>

</launch>
